src/Message/Timestamp.php

```diff
@@ -15,8 +15,8 @@ public function jsonSerialize(): string
 
     public function __toString(): string
     {
-        $me = clone $this;
```

Comment:
This is done to make sure that changes are always applied to a copy of the data. Immutability is one of the cornerstones of this library so please leave this in.